### PR TITLE
[Example] Update docs

### PR DESCRIPTION
**Pull request by zhoonit**
**Changes proposed in this PR:**
- Update docs
- Add demo footage
- Delete batch from model input shape

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

### CODE REVIEW

**Comment by zhoonit** [comment]
Please refer to https://github.com/zhoonit/nntrainer/tree/chores/docs/Applications/Tizen_native/CustomShortcut

**Review by jijoongmoon — APPROVED** [review]
LGTM

File: Applications/Tizen_native/CustomShortcut/README.md [hunk 1]
```diff
@@ -16,24 +18,28 @@ If you want to do in CLI, you first need to [convert the project to CLI](https:/
...
-$ sdb push capi-nntrainer.0.0.1-0.armv7l.rpm /usr
-$ popd
-$ sdb shell
-sh-3.2# mount -o remount,rw /
-sh-3.2# cd /usr
-sh-3.2# rpm -e nntrainer
-sh-3.2# rpm -i nntrainer.0.0.1.-0.armv7l.rpm
-sh-3.2# rpm -i capi-nntrainer.0.0.1.-0.armv7l.rpm
+$ sdb shell "mount -o remount,rw /"
+$ sdb push ${gbs repo directory} /tmp_repos/
```
Comment (kparichay) [inline Applications/Tizen_native/CustomShortcut/README.md:37]: I think now we dont need this as nntrainer comes with wearable image.
For some reason, capi-nntrainer is not part of the image.
@again4you can you give some hints why this is happening?

Comment (zhoonit) [reply Applications/Tizen_native/CustomShortcut/README.md:37]: You are right. I'll package capi and then update this line

**Pull request merged**